crates/ln-dlc-node/src/lib.rs

```diff
@@ -28,7 +28,7 @@ mod ln;
 mod ln_dlc_wallet;
 mod logger;
 mod on_chain_wallet;
-mod seed;
+pub mod seed;
```

Comment:
As discussed, perhaps the `ln-dlc-node` library shouldn't mandate what kind of seed to use. We could just let consumers provide a seed for LDK and another for the on-chain wallet.

In practice, I don't think it matters too much because we will want to use the same BIP39 seed for app and coordinator.